.php_cs.dist

```diff
@@ -19,6 +19,8 @@ $config = PhpCsFixer\Config::create()
         '@PHPUnit60Migration:risky' => true,
         'php_unit_dedicate_assert' => ['target' => 'newest'],
         'native_function_invocation' => true,
+        'no_alias_functions' => true,
```

Comment:
What about adding `@PhpCsFixer` and `@PHP71Migration` too?

Reply:
No clue what the effect will be but I suggest we do this in a follow up PR to keep PRs small.

Reply:
MR = PR? So you are ok to merge and then follow up? :-)